android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -301,8 +304,25 @@ class CustomTabsToolbarFeature(
         }
     }
 
+    private fun Bitmap.resizeMaintainingAspectRatio(): Size {
```

Comment:
Ideally this would have been in [org.mozilla.fenix.ext.Bitmap](https://github.com/mozilla-mobile/firefox-android/blob/7d0c8cacb416c627eec2c1bb97b7d13964736130/fenix/app/src/main/java/org/mozilla/fenix/ext/Bitmap.kt#L4) but this would introduce a dependency on Fenix app

Reply:
you could alternatively create a bitmap ext within `support-utils` if you feel this has value elsewhere

Reply:
Good suggestion 👍